ui/src/shared/containers/RouteToOrg.tsx

```diff
@@ -0,0 +1,46 @@
+// Libraries
+import {PureComponent} from 'react'
+import {connect} from 'react-redux'
+import {WithRouterProps} from 'react-router'
+
+// Types
+import {AppState} from 'src/types'
+import {Organization} from '@influxdata/influx'
+
+// Decorators
+
+interface StateProps {
+  orgs: Organization[]
+  org: {id?: string}
```

Comment:
org has id and name, I'm surprised this isn't causing linter error

Reply:
org has an optional id and name but this interface only requires it to have an optional id. Since its just an interface, it doesnt need to match exactly, just meet the expectations of the interface.